_posts/2020-09-21-changelog-131.markdown

```diff
@@ -0,0 +1,103 @@
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6039 -->
+* Provide intention to create a method from an unresolved method call (by [@Kobzol])
+
+    {% include gif-img.html path="/assets/posts/changelog-131/create_method" w="650px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5928 -->
+* Add initial parsing of [the new asm! macro](https://github.com/rust-lang/rfcs/pull/2873) (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5939 -->
+<!-- * RUN: Add build task for wasm-pack configurations -->
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5720 -->
+* Refactor checking and filling of match expressions
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5369 -->
+* Deprioritize completion of `&mut self` methods for const references and bindings (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5657 -->
+* Handle name conflicts after renaming local variables (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6054 -->
+<!-- * RUN: Add target completion for cargo command -->
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6013 -->
+<!-- * Don't run inspections on files outside a valid project -->
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5956 -->
+* Add `dbgr` postfix completion (by [@mgrachev])
+
+    {% include gif-img.html path="/assets/posts/changelog-131/dbgr" w="406px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6010 -->
+* Make moved items public if needed in move refactoring (`Refactor | Move` or <kbd>F6</kbd>)
+
+    {% include gif-img.html path="/assets/posts/changelog-131/move_make_items_public" w="373px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6057 -->
+* Introduce a quick fix to add `..` to struct pattern that has missing fields (by [@t-kameyama])
+
+    {% include gif-img.html path="/assets/posts/changelog-131/pat_rest" w="400px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5995 -->
+* Rename "Cargo Command" run configuration to "Cargo"
+
+    <img src="/assets/posts/changelog-131/cargo_run_configuration_name.png" alt="Cargo run configuration screenshot" width="648px"/>
+
+## Fixes
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6075 -->
+* Fix "File is not included in module tree" warnings for a project under a symlinked directory
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6033 -->
+* Don't resolve std macros when [`no_std`](https://rust-embedded.github.io/book/intro/no-std.html) is present (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6041 -->
+* Fix highlighting inside macro calls in some cases
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6049 -->
+* Fix flattening nested `use` statements intention (by [@t-kameyama])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6078 -->
+* Improve visibility check in move items refactoring
```

Comment:
Maybe it's worth to move this item into features section and create top-level item about `Move` refactoring?